frontend/src/components/CalcSpectrum.tsx

```diff
@@ -116,19 +116,25 @@ export const CalcSpectrum: React.FC = () => {
           .max(30, "Simulate slit must be less than 30"),
       }),
   });
-  const { control, handleSubmit, setValue, watch, formState } =
-    useForm<FormValues>({
-      defaultValues: { species: [{ molecule: "CO", mole_fraction: 0.1 }] },
-      resolver: yupResolver(Schema),
-    });
+  const { control, handleSubmit, setValue, watch } = useForm<FormValues>({
+    defaultValues: { species: [{ molecule: "CO", mole_fraction: 0.1 }] },
+    resolver: yupResolver(Schema),
+  });
 
-  console.log(formState?.errors);
   const handleBadResponse = (message: string) => {
     setCalcSpectrumResponse(undefined);
     setError(message);
   };
   const onSubmit = async (data: FormValues): Promise<void> => {
-    console.log(data);
+    if (useSlit == true) {
+      if (data.mode === "radiance_noslit") {
+        data.mode = "radiance";
+      }
+      if (data.mode === "transmittance_noslit") {
+        data.mode = "transmittance";
+      }
```

Comment:
We also need to update the label logic:
```ts
  let modeLabel;
  if (mode === "absorbance") {
    modeLabel = "Absorbance";
    data.units = "-ln(I/I0)";
  } else if (mode === "transmittance_noslit") {
    modeLabel = "Transmittance";
  } else if (mode === "radiance_noslit") {
    modeLabel = "Radiance";
  } else {
    throw new Error("Invalid mode");
  }
```

we could fix it by changing it to `mode.startsWith("radiance")`, etc.